examples/models/disruption_budgets/pdbs_example.ipynb

```diff
@@ -23,7 +23,7 @@
    "source": [
     "## Setup Seldon Core\n",
     "\n",
-    "Use the setup notebook to [Setup Cluster](../../../notebooks/seldon_core_setup.ipynb#Setup-Cluster) with [Ambassador Ingress](../../../notebooks/seldon_core_setup.ipynb#Ambassador) and [Install Seldon Core](../../seldon_core_setup.ipynb#Install-Seldon-Core). Instructions [also online](./seldon_core_setup.html)."
```

Comment:
I always worry when I see absolute links... One of the issues is that even if someone will be browsing docs for a given version link will lead to latest...

Reply:
I agree. Unfortunately, there isn't a better alternative when using `nbsphinx-link`.

Reply:
Maybe would be worth moving all examples / notebooks inside `doc/` folder at some point? Would that help?

Reply:
That would help, and that's kind of what nbsphinx favours. However, I believe it was decided not to do that to keep the examples folder at the root level of the repo.

Maybe an option worth exploring would be using sym links? Not sure how that would work with Git and nbsphinx.

Reply:
Symlinks seems reasonable, imo. Maybe worth following an follow up issue on that.

Reply:
Good shout. I'll open up an issue.

Reply:
For reference, this is the issue exploring the symlinks: https://github.com/SeldonIO/seldon-core/issues/2649